### DISCUSSION TITLE
Durexforth Benchmarks

### DISCUSSION THREAD

**Discussion by burnsauce** [jkotlinski/durexforth#414] [Show and tell]
From the most recent branch:
![image](https://user-images.githubusercontent.com/158256/148501578-3b8635a2-b5dc-4f7b-b4e6-95387cbde3f5.png)
Code for the tests
[bench.txt](https://github.com/jkotlinski/durexforth/files/7826941/bench.txt)

**Comment by jkotlinski** [comment]
To compare with results at https://www.theultimatebenchmark.org/

This is a solid improvement compared to 1.6.1 👍
